Testing/SystemTests/tests/analysis/SANSILLAutoProcessTest.py

```diff
@@ -63,6 +65,11 @@ def runTest(self):
                 OutputWorkspace='iq_s' + str(i + 1)
             )
 
+            try:
```

Comment:
You should rather update the references to cover also the stitched output.

Reply:
I changed the reference files when needed and removed the try-catch blocks.